gsoc/2021.md

```diff
@@ -0,0 +1,505 @@
+﻿---
+layout: inner-page-no-masthead
+title: "Google Summer of Code 2021 Scala Projects"
+permalink: /gsoc/2021/
+includeTOC: true
+---
+
+## Summary & goals
+
+[The Scala Center](https://scala.epfl.ch/) organisation and its team are dedicated to moving the needle towards a more diverse community, (amongst other) by providing easy-to-access first experience and introduction to the Scala ecosystem. GSoC is a fantastic opportunity to engage with students all around the world and onboard new generations, as well as underrepresented groups in tech!
+
+
+Scala is a part of Google Summer of Code since 2010. With 65 projects & students, ~70 mentors over the years, many of whom continued contributing to Scala, we are happy to announce that in 2021 the Scala Center will reignite the collaboration and commit to providing an awesome experience going forward.

```

Comment:
maybe:

> Scala has been a part of Google Summer of Code most years since 2010. In sum, about 65 students have been guided by our mentors and completed projects. Many participants continued contributing to Scala afterwards.

> But we weren't able to participate in 2019 or 2020. We are happy to announce that this year, the Center will reignite the collaboration and resume providing this awesome experience.